extensions/sns/e2e/tests/sns.bash

```diff
@@ -171,36 +149,9 @@ SNS_CONFIG_FILE_V2_NAME="sns_v2.yml"
 }
 
 # This test asserts that the `propose` subcommand exist in the current extension version.
-@test "sns propose exists" {
-    run dfx sns propose --help
-    assert_output --partial "dfx sns propose"
-}
-
-# This test asserts that at least one neuron flag must be specified to succeed
-@test "sns propose must use a neuron flag" {
-    install_asset sns/valid
-
-    run dfx sns propose sns_v2.yml
-    assert_failure
-    assert_output --partial "sns propose --dfx-cache-path <DFX_CACHE_PATH> <--neuron-id <NEURON_ID>|--neuron-memo <NEURON_MEMO>|--test-neuron-proposer> <INIT_CONFIG_FILE>"
-}
-
-# This test asserts that a local dfx server with the NNS installed can submit a
-# CreateServiceNervousSystem NNS Proposal with the test neuron
-@test "sns propose can submit a proposal with the test neuron" {
-    dfx_new
-
-    dfx_extension_install_manually nns
-    install_shared_asset subnet_type/shared_network_settings/system
-    install_asset sns/valid
-
-    dfx_start_for_nns_install
-    dfx nns install
-
-    run dfx sns propose --test-neuron-proposer "${SNS_CONFIG_FILE_V2_NAME}"
-    assert_success
-    assert_output --partial "🚀 Success!"
-    assert_output --partial "Proposal ID"
+@test "sns deploy-testflight exists" {
+    run dfx sns deploy-testflight --help
```

Comment:
Similar to an earlier comment: Can't you do something like dfx_start so that you don't need to do this "dry run" via `--help` ?

Reply:
I believe the other `exists` tests work by using `--help`